torch/jit/__init__.py

```diff
@@ -1312,6 +1312,11 @@ def forward(self, input):
     'pairwise_distance',
     'prelu',
     'hardshrink',
+    'dropout',
```

Comment:
It looks like we have dropout/alpha_dropout/feature_alpha_dropout as Aten ops, why we are adding them to the blacklist?

Reply:
When these are used from `torch.nn.functional` they should have the same semantics as the Python versions (which is mostly throwing exceptions on bad input / forwarding different args), so this makes them use the weak script versions from `torch.nn.functional` rather than just the aten op

Reply:
gotcha, shall we make the comment above be more clear so that we are not confused with it?